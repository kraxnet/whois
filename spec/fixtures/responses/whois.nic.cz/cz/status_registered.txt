%  (c) 2006-2017 CZ.NIC, z.s.p.o.
% 
% Intended use of supplied data and information
% 
% Data contained in the domain name register, as well as information
% supplied through public information services of CZ.NIC association,
% are appointed only for purposes connected with Internet network
% administration and operation, or for the purpose of legal or other
% similar proceedings, in process as regards a matter connected
% particularly with holding and using a concrete domain name.
% 
% Full text available at:
% http://www.nic.cz/page/306/intended-use-of-supplied-data-and-information/
% 
% See also a search service at http://www.nic.cz/whois/
% 
% 
% Whoisd Server Version: 3.10.2
% Timestamp: Thu Sep 07 12:49:39 2017

domain:       nic.cz
registrant:   CZ-NIC
admin-c:      FEELA
admin-c:      MAPET
nsset:        CZ.NIC
keyset:       CZNIC
registrar:    REG-CZNIC
status:       Sponsoring registrar change forbidden
registered:   30.10.1997 01:00:00
changed:      22.11.2016 15:07:40
expire:       15.03.2027

contact:      CZ-NIC
org:          CZ.NIC, z.s.p.o.
name:         CZ.NIC, z.s.p.o.
address:      Milesovska 1136/5
address:      Praha 3
address:      130 00
address:      CZ
phone:        +420.222745111
fax-no:       +420.222745112
e-mail:       admin@nic.cz
registrar:    REG-CZNIC
created:      17.10.2008 12:08:21
changed:      09.01.2015 11:09:33

contact:      FEELA
name:         Ondřej Filip
address:      Jeseniova 2852/16
address:      Praha 3
address:      13000
address:      CZ
registrar:    REG-MOJEID
created:      04.01.2005 13:35:00
changed:      17.07.2017 15:35:48

contact:      MAPET
name:         Martin Peterka
registrar:    REG-MOJEID
created:      06.03.2002 18:11:00
changed:      16.02.2017 19:42:41

nsset:        CZ.NIC
nserver:      d.ns.nic.cz (193.29.206.1, 2001:678:1::1)
nserver:      a.ns.nic.cz (194.0.12.1, 2001:678:f::1)
nserver:      b.ns.nic.cz (194.0.13.1, 2001:678:10::1)
tech-c:       JTALIR
tech-c:       JAROMIR-TALIR
registrar:    REG-CZNIC
created:      09.06.2008 14:30:16
changed:      20.09.2013 11:18:20

contact:      JTALIR
name:         Jaromír Talíř
registrar:    REG-MOJEID
created:      26.10.2010 10:53:34
changed:      15.11.2016 15:04:44

contact:      JAROMIR-TALIR
name:         Jaromír Talíř
address:      Americká 23
address:      Praha 2
address:      120 00
address:      CZ
registrar:    REG-MOJEID
created:      01.12.2011 17:35:04

keyset:       CZNIC
dnskey:       257 3 13 LM4zvjUgZi2XZKsYooDE0HFYGfWp242fKB+O8sLsuox8S6MJTowY8lBDjZD7JKbmaNot3+1H8zU9TrDzWmmHwQ==
tech-c:       JTALIR
registrar:    REG-CZNIC
created:      20.05.2016 16:13:26


