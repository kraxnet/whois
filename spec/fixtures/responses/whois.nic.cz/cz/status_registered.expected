#disclaimer
  %s == " (c) 2006-2017 CZ.NIC, z.s.p.o.\n\nIntended use of supplied data and information\n\nData contained in the domain name register, as well as information\nsupplied through public information services of CZ.NIC association,\nare appointed only for purposes connected with Internet network\nadministration and operation, or for the purpose of legal or other\nsimilar proceedings, in process as regards a matter connected\nparticularly with holding and using a concrete domain name.\n\nFull text available at:\nhttp://www.nic.cz/page/306/intended-use-of-supplied-data-and-information/\n\nSee also a search service at http://www.nic.cz/whois/\n\n"

#domain
  %s == "nic.cz"

#domain_id
  %s raise_error(Whois::AttributeNotSupported)

#status
  %s %CLASS{array}
  %s == [:server_transfer_prohibited]

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %CLASS{time}
  %s == Time.parse("30.10.1997 01:00:00")

#updated_on
  %s %CLASS{time}
  %s == Time.parse("22.11.2016 15:07:40")

#expires_on
  %s %CLASS{time}
  %s == Time.parse("15.03.2027")

#registrar
  %s %CLASS{registrar}
  %s.id           == "REG-CZNIC"
  %s.name         == "REG-CZNIC"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == "CZ-NIC"
  %s[0].name          == "CZ.NIC, z.s.p.o."
  %s[0].organization  == "CZ.NIC, z.s.p.o."
  %s[0].address       == "Milesovska 1136/5\nPraha 3\n130 00\nCZ"
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country_code  == nil
  %s[0].email         == "admin@nic.cz"
  %s[0].created_on    == Time.parse("17.10.2008 12:08:21")
  %s[0].updated_on    == Time.parse("09.01.2015 11:09:33")

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "FEELA"
  %s[0].name          == "Ondřej Filip"
  %s[0].organization  == nil
  %s[0].address       == "Jeseniova 2852/16\nPraha 3\n13000\nCZ"
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country_code  == nil
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == nil
  %s[0].created_on    == Time.parse("04.01.2005 13:35:00")
  %s[0].updated_on    == Time.parse("17.07.2017 15:35:48")

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0].id            == "JTALIR"
  %s[0].name          == "Jaromír Talíř"
  %s[1] %CLASS{contact}
  %s[1].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[1].id            == "JAROMIR-TALIR"
  %s[1].name          == "Jaromír Talíř"
  %s[1].organization  == nil
  %s[1].address       == "Americká 23\nPraha 2\n120 00\nCZ"
  %s[1].city          == nil
  %s[1].zip           == nil
  %s[1].state         == nil
  %s[1].country_code  == nil
  %s[1].phone         == nil
  %s[1].fax           == nil
  %s[1].email         == nil
  %s[1].created_on    == Time.parse("01.12.2011 17:35:04")
  %s[1].updated_on    == nil

#nameservers
  %s %CLASS{array}
  %s %SIZE{3}
  %s[0] %CLASS{nameserver}
  %s[0].name == "d.ns.nic.cz"
  %s[0].ipv4 == "193.29.206.1"
  %s[0].ipv6 == "2001:678:1::1"
  %s[1] %CLASS{nameserver}
  %s[1].name == "a.ns.nic.cz"
  %s[2] %CLASS{nameserver}
  %s[2].name == "b.ns.nic.cz"
